packages/mantine/src/components/modal-wizard/ModalWizard.tsx

```diff
@@ -102,15 +77,17 @@ export const ModalWizard: ModalWizardType = ({
     nextButtonLabel = 'Next',
     previousButtonLabel = 'Previous',
     finishButtonLabel = 'Finish',
+    title,
```

Comment:
eslint says title is not used? if so can you remove it? 😆

Reply:
I removed `title` from here, also I omitted this prop from ModalProps here: 
```
interface ModalWizardProps extends Omit<ModalProps, 'centered'>{}
```